connectors/http/http-base/src/main/java/io/camunda/connector/http/base/model/CommonRequest.java

```diff
@@ -25,22 +26,23 @@
 
 public class CommonRequest {
 
+  @FEEL
   @NotBlank
-  @Pattern(regexp = "^(http://|https://|secrets|\\{\\{).*$")
+  @Pattern(regexp = "^(=|http://|https://|secrets|\\{\\{).*$")
   private String url;
 
-  @NotBlank private String method;
+  @FEEL @NotBlank private String method;
 
   @Valid private Authentication authentication;
 
   @Pattern(regexp = "^([0-9]+|secrets\\..+)$")
   private String connectionTimeoutInSeconds;
```

Comment:
We can map it to an Integer/Long now as its supported. We can also remove the pattern as its outdated.

Reply:
@Oleksiivanov We should support it as we use jackson to bind the values which allows to map from `"10"` to `10`.